src/server/config.rs

```diff
@@ -213,6 +219,8 @@ impl Default for Config {
             raft_client_queue_size: 8192,
             raft_msg_max_batch_size: 128,
             grpc_compression_type: GrpcCompressionType::None,
+            grpc_gzip_compression_level: 6,
```

Comment:
Better to use  DEFAULT_GRPC_GZIP_COMPRESSION_LEVEL and DEFAULT_GRPC_COMPRESSION_LOWER_BOUND.